cfdp-core/src/transaction/config.rs

```diff
@@ -7,11 +7,29 @@ use crate::{
     filestore::ChecksumType,
     pdu::{
         CRCFlag, Condition, EntityID, FaultHandlerAction, FileSizeFlag, FileStoreRequest,
-        MessageToUser, SegmentedData, TransactionSeqNum, TransmissionMode,
+        MessageToUser, SegmentedData, TransactionSeqNum, TransmissionMode, VariableID,
     },
 };
 
-pub type TransactionID = (EntityID, TransactionSeqNum);
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub struct TransactionID(pub EntityID, pub TransactionSeqNum);
```

Comment:
I don't mind this being a named tuple, that's an easy conversion with existing code.
I could imagine using name fields too if we want access to be less ambiguous for the reader.

Reply:
@xpromache any thoughts here? I like to merge this before #30.

Reply:
For me it's ok like it is, I don't feel an urgency to change it to use named fields. I could change it if you want but I don't have time right now, maybe in 4-5 hours.

Reply:
This is fine for me for now. We can always change it later too.